engine/src/test/scala/cromwell/webservice/CromwellApiServiceSpec.scala

```diff
@@ -588,9 +643,23 @@ object CromwellApiServiceSpec {
   class MockServiceRegistryActor extends Actor {
     import MockServiceRegistryActor._
     override def receive = {
-      case WorkflowQuery(_) =>
+      case WorkflowQuery(parameters) =>
+        val labels: Option[Map[String, String]] = {
+          if (parameters.contains(("additionalKeys", "labels"))) {
```

Comment:
Hmm, I'm quite possibly misreading this but it looks like this is saying, 
> "if my test `GET` wants labels then my tests adds labels to the result it returns, and then my test checks that it added the labels"

As far as I can tell, this doesn't actually test that the underlying logic is really adding labels when it needs to?

Reply:
How do you suggest modifying the test? Previously, this was always returning the same `WorkflowQueryResponse` no matter what the query was.